docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -54,7 +54,7 @@ If you are using a guide involving Salt, see [Configuring Kubernetes with Salt](
 
 * [Using Admission Controllers](/docs/admin/admission-controllers/) explains plug-ins which intercepts requests to the Kubernetes API server after authentication and authorization.
 
-* [Using Sysctls in a Kubernetes Cluster](/docs/concepts/cluster-administration/sysctl-cluster/) describes to an administrator how to use the `sysctl` command-line tool to set kernel parameters .
+* [Using Sysctl in a Kubernetes Cluster](/docs/concepts/cluster-administration/sysctl-cluster/) describes to an administrator how to use the `sysctl` command-line tool to set kernel parameters .
```

Comment:
I don't think this is a valid change. The 'sysctls' here meant the various knobs an admin can tune, it is not about the command line tool.

Reply:
yes, I didn't understand this well, thanks for your correct :P

Reply:
Agree with @tengqm.